L1Trigger/CSCTriggerPrimitives/python/cscTriggerPrimitiveDigis_cfi.py

```diff
@@ -11,7 +11,7 @@
 from L1Trigger.CSCTriggerPrimitives.params.tmbParams import tmbPSets
 from L1Trigger.CSCTriggerPrimitives.params.auxiliaryParams import auxPSets
 from L1Trigger.CSCTriggerPrimitives.params.gemcscParams import gemcscPSets
-from L1Trigger.CSCTriggerPrimitives.params.showerParams import showerPSet,showerPSet_2025
+from L1Trigger.CSCTriggerPrimitives.params.showerParams import showerPSet,showerPSet_2025_AtoE,showerPSet_FtoG
```

Comment:
This `import` asks for `showerPSet_FtoG`, but the `showerParams.py` defines `showerPSet_2025_FtoG` (note `_2025` in the latter that is not in the former).

Reply:
Thanks for the spot Matti: @AlexAlbert41 can you fix the typo here please.